### DISCUSSION TITLE
is it possible to customize the nav bar? or theme it i any way.

### DISCUSSION THREAD

**Discussion by bmirandan** [SAP/luigi#2959] [Q&A]
Is it possible to change the navbar?  theme it or put a custom one? The idea is to provide navigation across the micro front ends by using it or you could only achieve this by adding a micro-frontend with the navbar and using luigi navigation?

**Comment by alexandra-simeonova** [comment]
Hi @bmirandan. Yes, it is possible to customize Luigi. See the FAQ question[ "I don't want to use the default Fiori Fundamentals style. How can I style Luigi differently? How can I change the look of Luigi's UI?" ](https://docs.luigi-project.io/docs/faq?section=ui-questions)
